setup.py

```diff
@@ -7,53 +7,58 @@
 from setuptools import find_packages
 
 install_requires = [
-    'setuptools',
-    'six',
-    'Jinja2',
+    "setuptools",
+    "six",
```

Comment:
I am not behind my laptop, so I cannot check: is six still needed? It should not be.

Reply:
I now removed six as well